pkg/agent/openflow/pipeline.go

```diff
@@ -41,38 +41,126 @@ import (
 )
 
 var (
-	ClassifierTable              = binding.NewOFTable(0, "Classification")
-	SpoofGuardTable              = binding.NewOFTable(10, "SpoofGuard")
-	arpResponderTable            = binding.NewOFTable(20, "ARPResponder")
-	IPv6Table                    = binding.NewOFTable(21, "IPv6")
-	MulticastTable               = binding.NewOFTable(22, "Multicast")
-	ServiceHairpinTable          = binding.NewOFTable(23, "ServiceHairpin")
-	ServiceConntrackTable        = binding.NewOFTable(24, "ServiceConntrack") // serviceConntrackTable use a new ct_zone to transform SNAT'd connections.
-	ConntrackTable               = binding.NewOFTable(30, "ConntrackZone")
-	ConntrackStateTable          = binding.NewOFTable(31, "ConntrackState")
-	ServiceClassifierTable       = binding.NewOFTable(35, "ServiceClassifier")
-	SessionAffinityTable         = binding.NewOFTable(40, "SessionAffinity")
-	DNATTable                    = binding.NewOFTable(40, "DNAT(SessionAffinity)")
-	ServiceLBTable               = binding.NewOFTable(41, "ServiceLB")
-	EndpointDNATTable            = binding.NewOFTable(42, "EndpointDNAT")
-	AntreaPolicyEgressRuleTable  = binding.NewOFTable(45, "AntreaPolicyEgressRule")
-	DefaultTierEgressRuleTable   = binding.NewOFTable(49, "DefaultTierEgressRule")
-	EgressRuleTable              = binding.NewOFTable(50, "EgressRule")
-	EgressDefaultTable           = binding.NewOFTable(60, "EgressDefaultRule")
-	EgressMetricTable            = binding.NewOFTable(61, "EgressMetric")
-	L3ForwardingTable            = binding.NewOFTable(70, "L3Forwarding")
-	SNATTable                    = binding.NewOFTable(71, "SNAT")
-	L3DecTTLTable                = binding.NewOFTable(72, "IPTTLDec")
-	L2ForwardingCalcTable        = binding.NewOFTable(80, "L2Forwarding")
-	AntreaPolicyIngressRuleTable = binding.NewOFTable(85, "AntreaPolicyIngressRule")
-	DefaultTierIngressRuleTable  = binding.NewOFTable(89, "DefaultTierIngressRule")
-	IngressRuleTable             = binding.NewOFTable(90, "IngressRule")
-	IngressDefaultTable          = binding.NewOFTable(100, "IngressDefaultRule")
-	IngressMetricTable           = binding.NewOFTable(101, "IngressMetric")
-	ConntrackCommitTable         = binding.NewOFTable(105, "ConntrackCommit")
-	ServiceConntrackCommitTable  = binding.NewOFTable(106, "ServiceConntrackCommit")
-	HairpinSNATTable             = binding.NewOFTable(108, "HairpinSNAT")
-	L2ForwardingOutTable         = binding.NewOFTable(110, "Output")
+	// PipelineClassifierTable is table 0. Packets are forwarded to different pipelines in this table. Note that, the value
```

Comment:
@jianjuns @tnqn From line 44 to line 163 is the newly added comments. Thanks for reviewing.